Source/Public/Publish-Bicep.ps1

```diff
@@ -0,0 +1,46 @@
+function Publish-Bicep {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory, Position = 1)]
+        [ValidateNotNullOrEmpty()]
+        [string]$Path,
```

Comment:
Should we have a `[ValidateScript({Test-Path $_})]` here to make sure the file exists?